package.json

```diff
@@ -60,6 +60,7 @@
     "ci:test:core:components": "yarn wsrun --serial --exclude-missing ci:test",
     "ci:test:frontend": "yarn workspace blockchain-wallet-v4-frontend ci:test:frontend",
     "clean": "cross-env yarn wsrun clean && rimraf build && rimraf coverage && rimraf dist && rimraf *.log && rimraf node_modules",
+    "commit": "git cz --gpg-sign",
```

Comment:
custom commit cmd? please remove

Reply:
I added it at [the suggestion of the Commitizen documentation](https://github.com/commitizen/cz-cli#if-your-repo-is-commitizen-friendly).

Do you not think it's useful for making it easier for developers to comply with Commitizen?